File: frontend/src/lib/supabase.ts
import { VITE_PUBLIC_SUPABASE_ANON_KEY, VITE_SUPABASE_URL } from "@/constants";
import { createClient } from "@supabase/supabase-js";

const Supabase = createClient(VITE_SUPABASE_URL, VITE_PUBLIC_SUPABASE_ANON_KEY);

export default Supabase

File: backend/libs/supabase.py
from supabase._async.client import AsyncClient, create_client as create_async_client
from backend.settings import Settings  # type: ignore

settings = Settings()


async def get_supabase_async_client() -> AsyncClient:
    return await create_async_client(
        settings.SUPABASE_URL, settings.SUPABASE_SERVICE_KEY
    )
